### PR TITLE
Core style

### PR DESCRIPTION
**Pull request by Hector096**
Milestone 1: Add Core style in the app

### CODE REVIEW

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
### Hi Team 😃 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that lacks the correct linters setup. Because you are now in the javascript module there is no need for the lighthouse linter check. This might have been missed by the other reviewers in the previous milestones but you are required to kindly correct this. 🙏 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -1,7 +1,45 @@
...
+form .btn:hover {
+  background-color: #0d6efd;
+  color: white;
+}
+
+/* Start List */
+.books-container {
+  max-height: 300px;
+  overflow-y: scroll;
+}
```
Comment (george-shammar) [inline style.css:18]: - [ ] While you free to use styles of your choice (colors, fonts, backgrounds, etc.), you are required to follow the general layout as much as possible. Please see below differences between your page presentation and the wireframe template.

Your design: Please specify the book is by the author as provided in the wireframe design. It makes a good user experience. You don't want your users getting confused :smile: 

Screenshot 2021-09-16 at 09-34-17 Awesome Books

Wireframe template.
Screenshot_2021-09-16_09-38-26

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!

:heavy_check_mark: Your design looks cool. :+1: 

There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @Hector096 and @AhmedOmarAL-farouq 

May I highlight the following:

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- You have a descriptive Readme file:heavy_check_mark:
- The live link is working properly:heavy_check_mark:
- Your project looks awesome:heavy_check_mark:

## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewers. 
 
 #### Optional suggestions
    
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**